client/src/components/DrawingScreen/DrawingScreen.test.jsx

```diff
@@ -0,0 +1,38 @@
+// Link.react.test.js
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import axios from 'axios';
+import DrawingScreen, { Point, Stroke } from './DrawingScreen';
+import '../../test/setupTests';
+
+describe('DrawingScreen', () => {
+  const mockGameState = { groupName: 'kitties4Life', currentPlayer: { name: 'baby cat' } };
+  const mockOnGameStateChanged = jest.fn();
+
+  it('renders strokes as a data URL', () => {
```

Comment:
why strokes as url? is that a common pattern? 

i dont have any alternative suggestions - just curious

Reply:
Well I think we have 2 options:
- Store strokes on the server (gets VERY chunky with complex drawings)
- Store images on the server (more or less same chunkdom)

Canvas elements have a method to render the contents as an image in the format of a data url that you could set the arc of an image to so I thought that'd be a good candidate for transmitting the rendered image